### PR TITLE
Documenting V7 integration

### PR DESCRIPTION
**Pull request by brimoor**

### CODE REVIEW

**Comment by allenleetc** [comment]
Have small correction and will test further on schema/attribs, fill in TODO note etc

**Comment by brimoor** [comment]
@allenleetc yes thank you! I was just copy + pasting so if you could remove anything unsupported, fix any bugs, etc. that would be terrific

File: docs/source/integrations/v7.rst [hunk 1]
```diff
@@ -0,0 +1,968 @@
...
+or by adding the following parameters to your
+:ref:`annotation config <annotation-config>` located at
+`~/.fiftyone/annotation_config.json`:
+
+.. code-block:: text
+
+    {
+        "backends": {
+            "darwin": {
+                "config_cls": "darwin_fiftyone.DarwinBackendConfig",
```
Comment (brimoor) [inline docs/source/integrations/v7.rst:208]: @allenleetc I seem to have written both `fiftyone_darwin` and `darwin_fiftyone`, did I mess this up?

File: docs/source/integrations/v7.rst [hunk 2]
```diff
@@ -0,0 +1,968 @@
...
+.. code-block:: shell
+
+    pip install fiftyone_darwin
+
+and register the ``darwin`` backend with FiftyOne, which you can do either by
+setting the following environment variables:
+
+.. code-block:: shell
+
+    export FIFTYONE_ANNOTATION_BACKENDS=*,darwin
```
Comment (allenleetc) [inline docs/source/integrations/v7.rst:195]: Is this syntax (with the asterisk) supported literally to mean eg keep default backends" ?

File: docs/source/integrations/v7.rst (hunk 1 above)
Comment (allenleetc) [reply docs/source/integrations/v7.rst:208]: Ah yes, will fix in a working PR for this

File: docs/source/integrations/v7.rst (hunk 2 above)
Comment (brimoor) [reply docs/source/integrations/v7.rst:195]: Yeah I just recently added support for this syntax here: https://github.com/voxel51/fiftyone/commit/6ab2ac970c956a16555ee222a2266bb2dccaa7a7

Unfortunately it won't be available until `fiftyone==0.24` is released though, so you should probably revert to explicitly listing the builtin backends for now

**Comment by brimoor** [comment]
Closing in favor of https://github.com/voxel51/fiftyone/pull/3910

**Pull request closed without merging**